app/src/main/java/cm/aptoide/pt/install/installer/DefaultInstaller.java

```diff
@@ -105,7 +98,10 @@ public PackageManager getPackageManager() {
           installation.setType(Installed.TYPE_UNKNOWN);
           moveInstallationFiles(installation);
         })
-        .flatMap(installation -> isInstalled(md5).first()
+        .flatMap(installation -> Observable.just(
+            isInstalled(installation.getPackageName(), installation.getVersionCode()))
+            .onErrorReturn(throwable -> false)
```

Comment:
Same thing as before about onErrorReturn

Reply:
This onErrorReturn should be here because it is related to the method above. If  the isInstalled returns an error we want it to pass false to the layers bellow.